test/Microsoft.DotNet.Cli.Utils.Tests/GivenAProjectDependenciesCommandFactory.cs

```diff
@@ -147,5 +147,36 @@ public void It_resolves_desktop_apps_using_configuration_passed_to_create()
             command.CommandName.Should().Contain(Path.Combine(testInstance.TestRoot, "bin", configuration));
             Path.GetFileName(command.CommandName).Should().Be("dotnet-desktop-and-portable.exe");
         }
+
+        [WindowsOnlyFact]
```

Comment:
?? Why would we only want this to run on windows?  It seems that this functionality should work across all platforms.